html-forms/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      ul {
+        padding: 0;
+      }
+      li {
+        list-style-type: none;
+      }
+    </style>
+  </head>
+  <body>
+    <form action="" method="post">
+      <fieldset>
+        <h1>Coding Form!</h1>
+        <ul>
+          <li>
+            <span style="display: block"><label for="mail">Email:</label></span>
+            <input type="email" id="mail" name="user_email" />
+          </li>
+          <li>
+            <span style="display: block">
+              <label for="question">Why is coding cool?</label></span
+            >
+            <select name="question" id="question">
+              <option value="select an option">selection an option</option>
+              <option value="because its interesting">
+                because its interesting
+              </option>
+              <option value="because its fun">because its fun</option>
+              <option value="because its powerful">because its powerful</option>
+            </select>
+          </li>
+        </ul>
+
+        <div>
+          <h3>Favorite Language (choose one)</h3>
+          <label for="html">HTML</label>
+          <input type="radio" name="language" id="html" value="html" />
+          <label for="css">CSS</label>
+          <input type="radio" name="language" id="css" value="css" />
+          <label for="javascript">Javascript</label>
+          <input
+            type="radio"
+            name="language"
+            id="javascript"
+            value="javascript" />
+        </div>
+
+        <div>
+          <h3>would you like to sign up for more information?</h3>
+          <input type="checkbox" id="check" name="signup" value="yes" />
+          <label for="check">Sign me up!</label>
+        </div>
+        <div>
+          <span style="display: block">message:</span>
+          <textarea name="comment" id="text" cols="30" rows="10"></textarea>
```

Comment:
Your `span` on 61 should be a `label` and it should be associated with your `textarea`

Reply:
would this be better? I thought could only be used once inside a form element.